ui/src/themes/extradark.js

```diff
@@ -0,0 +1,35 @@
+import blue from '@material-ui/core/colors/blue'
+
+export default {
+  themeName: 'Extra Dark',
+  palette: {
+    background: {
+      paper: '#000000',
+      default: '#000000',
+    },
+    primary: {
+      main: '#0f60b6',
+      contrastText: '#909090',
+    },
+    secondary: blue,
+    extra: {
+      bordercolors: '#202020',
+    },
+    type: 'dark',
+    extraAttribute: {
+      theme: 'extradark',
+      lines: '#353535',
```

Comment:
Looks like this attribute is not used.

Reply:
yeah I have just tried to change the border and lines under text color.